source/_lovelace/vertical-stack.markdown

```diff
@@ -21,24 +19,11 @@ type:
   type: string
 cards:
   required: true
-  description: List of cards
+  description: List of cards.
   type: list
-  keys:
-    type:
-      required: true
-      description: The type of the card to stack.
-      type: string
-    entity:
-      required: true
-      description: "An `entity_id` to use in the frontend."
-      type: string
-    image:
-      required: true
-      description: The URL to an image.
-      type: string
```

Comment:
Those keys are still present in the example below. Not documenting them will at some point lead to confusion as they seems to be required.

Reply:
These keys don't belong to this card. Instead they are part of the sub-cards